projects/freeradius/Dockerfile

```diff
@@ -15,17 +15,22 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
+
+# Run update a first time to get latest package definitions
+RUN apt-get update
```

Comment:
It's usually better to do this in the same step you are doing apt-get install. Otherwise caching can bite you because this layer can be much older than the install layer, causing it to fail.